src/main/resources/anniv5Resources/localization/eng/Potionstrings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "${ModID}:BoosterBrew": {
+    "NAME": "Booster Brew",
+    "DESCRIPTIONS": [
+      "Choose 1 of 3 random cards from different Packs. Add #b1 copy of it to your hand. It costs 0 this turn.",
+      " Repeat this effect #b",
+      " times."
+    ]
+  },
+  "${ModID}:ModdersDelight": {
+    "NAME": "Modder's Delight",
+    "DESCRIPTIONS": [
+      "Return all cards in your discard pile to your hand. Draw #b",
```

Comment:
The text and the effect for this don't line up -- looking at the code, it shuffles the discard pile back into your draw pile, then draws. Is that the intended effect? What I see the code doing makes more sense to me than what's written here.

If so this can use the wording of Deep Breath combined with the wording Booster Tutor: `Shuffle your discard pile into your draw pile. NL Put #bX random cards from different Packs from your draw pile into your hand.`